site/src/pages/TaskPage/TaskPage.tsx

```diff
@@ -386,3 +405,9 @@ function selectAgent(task: Task) {
 
 	return agents.at(0);
 }
+
+function selectChatApp(task: Task) {
+	return getTaskApps(task).find(
```

Comment:
it looks like this is the only place using `getTaskApps` now. what if instead of exposing it like this we left `getTaskApps` as it was (with the filter) and exported a `getTaskChatApp` function that does what this does?

Reply:
`getTaskApps` is also used elsewhere to fetch web apps, but I tried to simplify it by removing unnecessary function encapsulations.